.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
   - TEST_DOCTESTS="true" FASTCACHE="false" TEST_SETUP="true"
   global:
     - secure: "YIEZal9EBTL+fg2YmoZoS8Bvt3eAVUOZjb38CtqpzR2CCSXWoUk35KG23m2NknlY1iKfYJyt7XWBszT/VKOQEbWQq7PIakV4vIByrWacgBxy1x3WC+rZoW7TX+JJiL+y942qIYbMoNMMB8xFpE5RDLSjSecMpFhJJXoafVTvju8="
+    - TF_XLA_FLAGS=--tf_xla_cpu_global_jit
```

Comment:
What is the purpose of this? Does the module work if this variable isn't set? Can it be set in Python? If this is required for tensorflow to work, we should document it. Otherwise the module will work on Travis, but won't work anywhere else unless someone knows to set this environment variable.

Reply:
It raises a one time warning, if not set. I have given the detailed warning in the description of the PR. By the way, I believe that it would have been mentioned in tensorflow docs. Do we still need to document it? I just set it so that logs don't have unnecessary warnings. If it is not required, then please remove it.

Reply:
I see. Do you know what test is producing that? I don't really understand what the warning is talking about.

At any rate you should add a comment here explaining that this is to silence a warning in the tensorflow tests. It may be better to move it to the test_travis script, right before the optional dependency tests are called.

Reply:
In fact,  I also followed the instructions given in the warning. Actually,  I am away from my system right now. I will update the PR soon. As this is not critical,  so I will also look up into the details of the warning, it seems interesting.

Reply:
I read about the `XLA` at https://www.tensorflow.org/xla/overview. AFAICT, XLA is Accelerated Linear Algebra. The point is that XLA is in active development, so I believe that we should not use XLA for now. In fact, the warning is not that serious, so we can ignore that during the build.
Till now, I haven't figured out, how to silence that via code. By default, tensoflow is not compiled via XLA.
I will remove `TF_XLA_FLAGS=--tf_xla_cpu_global_jit` if no objection raised for 24 hours.

Reply:
That seems fine. There are also some other warnings printed during the tests from external dependencies that are not so easy to get rid of. https://github.com/sympy/sympy/pull/14747

Reply:
I opened https://github.com/tensorflow/tensorflow/issues/30308 for getting to any trick to silence the warning but received no response. So, I am finally removing the `TF_XLA_FLAGS=--tf_xla_cpu_global_jit`. 
Probably after that this PR will be done from my side and for adapting to new API of TF, I have opened #17149 and will draft a PR  soon on the basis of that.